### DISCUSSION TITLE
Unable to open primary script: /var/www/html/public/index.php (Operation not permitted)

### DISCUSSION THREAD

**Discussion by jericopulvera** [serversideup/docker-php#96] [Q&A] [answered]
This is a continuation of #93.

I'm able to deploy to ECS with v1.5.0 using the Fly Laravel template we just had an issue with Livewire and Media Library 
I'm trying to disable open_basedir due to it having an issue for some reason with Livewire and Media Library.

```
_file(): open_basedir restriction in effect. File(/media/image/product/a87ff679a2f3e71d9181a67b7542122c/image/59a3adea76fadcb6dd9e54c96fc155d1/photo-1505740420928-5e560c06d30e.jpeg) is not within the allowed path(s): (/var/www/html:/dev/stdout:/tmp)
```
I'm now trying to use v2.0.2 as that's the one with the variable to disable open_basedir and I have set it to none.

now I'm receiving this issue.

<img width="1878" alt="image" src="https://user-images.githubusercontent.com/23246308/201970374-0393bed7-3f23-408f-a391-0711e6537397.png">

I have made few changes to the fly template Dockerfile

from
```bash
cp .fly/nginx-default /etc/nginx/sites-enabled/default; \
```

to
```
rm /etc/nginx/sites-available/ssl-off; cp .fly/nginx-default /etc/nginx/sites-available/ssl-off; \
```

I have set the SSL_MODE mode to off in order to make this work.

**Comment by jericopulvera** [comment]
More logs

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/23246308/201971185-885c2841-07aa-44ff-b22f-142fc202b9f5.png">

**Comment by jaydrogers** [comment]
I am going to ping @fideloper on this one too since he is part of the team at Fly.

I know there were some issues with my latest images on Fly because of the S6 Overlay v3.x upgrade. I think there were some deeper stack issues with how S6 wanted to work and how Fly wanted things to work.

Maybe @fideloper could shine some light on this?

Let me know if there are any optimizations I can make to make my images better compatible with Fly 👍

**Comment by jericopulvera** [comment, marked as answer]
Sorry not really an issue with the Fly. I'm able to fix the issue by setting the PHP_OPEN_BASEDIR to / instead of none

I was able to debug it by trying it on a single index.php that outputs phpinfo(); inside.

I'm curious on what's the correct value in order to disable the OPEN_BASEDIR.

**Reply by jericopulvera** [reply]
Just a note. I'm just using the Fly template as a basis for the Laravel deployment but I'm deploying it on AWS ECS using AWS Copilot.

I haven't tested deploying v2.0.2 using Fly.

**Reply by jaydrogers** [reply]
It looks like you can set your value to `none` to disable it. More here: https://www.php.net/manual/en/ini.core.php#ini.open-basedir

**Reply by jericopulvera** [reply]
> It looks like you can set your value to `none` to disable it. More here: https://www.php.net/manual/en/ini.core.php#ini.open-basedir

Thats what I thought too. But setting it to none causes an error so I changed it to / and its now working. 

I have tried it in a build with only index.php to verify.

**Reply by jericopulvera** [reply]
Maybe I should try empty string instead of none string.
